src/controllers/forgotpassword.js

```diff
@@ -0,0 +1,54 @@
+import jwt from 'jsonwebtoken'
+import { Employee } from '../../models'
+import nodemailer from 'nodemailer'
+
+// Setting a transporter
+const transporter = nodemailer.createTransport({
+  // host and port for MailHog
+  host: '0.0.0.0',
+  port: 1025
+})
```

Comment:
These configurations also need to be variables... Remember that we don't expose sensitive data and we use environment variables for dynamic data that need to be changed in production.

Reply:
Well addressed, your can verify and approve the changes!